products/ocp4/profiles/cis.profile

```diff
@@ -113,6 +113,7 @@ selections:
     - api_server_etcd_ca
   # 1.2.33 Ensure that the --encryption-provider-config argument is set as appropriate
   # 1.2.34 Ensure that encryption providers are appropriately configured
+  # 2.8 Encrypt etc
```

Comment:
Looks good - but for some reason control 2.8 was removed in version 1.4.0 of the benchmark.. We'll just have to update this later (or let it disappear if we decide to use the new control file format).